src/Annotator.js

```diff
@@ -417,6 +428,7 @@ class Annotator extends EventEmitter {
      */
     renderPage(pageNum: number) {
         Object.keys(this.modeControllers).forEach((mode) => this.modeControllers[mode].renderPage(pageNum));
+        this.resetPopoverUI(pageNum);
```

Comment:
Again, `render` and `renderPage` execute this code at different times. Is the order important or can they be standardized? If not, let's leave a comment explaining why.